docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/header/header.scss

```diff
@@ -87,3 +87,18 @@
     }
   }
 }
+
+.toolbar-tray-open.toolbar-fixed {
```

Comment:
this is Drupalism (because the toolbar tray is from Drupal) and should be leaving outside of the library component. Could you please move it to `civictheme/assets/sass/header/_header.scss`.

Separately, could you please check if CivicTheme's mobile header is sticky and if it is - apply this to mobile header.

Thank you

Reply:
Moved the code to assets folder s per structure mentioned. 
Confirmed CivicTheme's mobile header is not sticky.